### PR TITLE
Enable supply chain for 4.9..4.11

### PR DESCRIPTION
**Pull request by akram**
Enables supply chain check for older versions and branches

### CODE REVIEW

File: ci-operator/config/openshift/jenkins/openshift-jenkins-release-4.12.yaml [hunk 1]
```diff
@@ -79,7 +79,7 @@ tests:
   commands: hack/check-plugins-supply-chain-change.sh
   container:
     clone: true
-    from: ocp_builder_rhel-8-golang-1.17-openshift-4.10
+    from: ocp_builder_rhel-8-golang-1.17-openshift-4.11
```
[the remark this answers is not in the document]
Comment (akram) [reply ci-operator/config/openshift/jenkins/openshift-jenkins-release-4.12.yaml:82]: correct @gabemontero

**Comment by akram** [comment]
> I'm assuming you'll be immediately creating 4.9 and 4.11 PRs to contain `make plugins-list` output and get the sorting correct @akram ? That is of course why the rehearsals failed.

yes, the 4.10 one has just been merged and 4.9 is pending

**Comment by gabemontero** [comment]
you are going to have to fix the ordered-prow-config e2e before this can merge @akram 

according to the log output you need to run `make jobs` and then commit the changes, update the PR

**Comment by gabemontero** [comment]
/hold

until @akram runs `make jobs` and fixes `ordered-prow-config`

**Comment by akram** [comment]
@gabemontero can you ltgm this one?
/retest

**Pull request merged**